src/tasks/validate-package.js

```diff
@@ -54,6 +58,25 @@ module.exports = function validatePackage(dir, functionHandler, restApiModule) {
 						throw routeMessage + 'error template requests custom headers but does not provide defaults';
 					}
 				}
+				if (methodConfig.error && methodConfig.error.defaultCode && !validHttpCode(methodConfig.error.defaultCode)) {
```

Comment:
I think this should just be error.code, not error.defaultCode, to keep things backwards compatible